src/Entity/Sip2User.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\Sip2UserRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=Sip2UserRepository::class)
+ */
+class Sip2User
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $username;
+
```

Comment:
A sip2 user consists of a username, a password and a location (=school). I think the school/location part is covered by the school enitity, but we are missing the password.

Reply:
I just disagreed with myself, we need the location on the sip2 user as well, as the "school" can have a different name (or want to show a different name on the screen) than what is used by FBS.

Reply:
But the school entity only has the "common name", if it is part of the SIP2 login i would expect a formalized `school code` or similar?

Reply:
Have added `$sip2InstitutionId` to `School` and `$password` to `Sip2User`